app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RailsReactWebpackIntro</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
```

Comment:
<img width="1514" alt="image" src="https://github.com/NicolasAndrehh/Rails-React-Webpack-Intro/assets/27709832/b00c23a1-daec-474c-a94f-310bf4da500e">

I keep getting the error attached above, kindly fix it.

Reply:
Hey @brytebee, I just cloned my project and I figured out that you're right, I'm getting the same error but I solve it just running the following commands:
```shh 
rails assets:precompile
```
And this one for populate the database with the greetings:
```shh 
rails db:migrate
```
Could you try it, please?

Reply:
- [ ] Well resolved.
<img width="318" alt="Screenshot 2023-07-11 at 18 08 50" src="https://github.com/NicolasAndrehh/Rails-React-Webpack-Intro/assets/58692590/5f40c1cd-8ed2-4c74-87e1-415efc83462c">